src-python/lab-service/traffexam/kilda/traffexam/action.py

```diff
@@ -20,6 +20,7 @@
 import nsenter
 from scapy.all import Ether, sendp
 from scapy.contrib import lldp
+from scapy.layers.l2 import ARP
```

Comment:
I think it would be better to either take `ARP` from the `.all` package as `Ether`, or explicitly specify `.layers.l2` for `Ether` and `sendp`.

Reply:
fixed